numpy/compat/py3k.py

```diff
@@ -7,23 +7,21 @@
 __all__ = ['bytes', 'asbytes', 'isfileobj', 'getexception', 'strchar',
            'unicode', 'asunicode', 'asbytes_nested', 'asunicode_nested',
            'asstr', 'open_latin1', 'long', 'basestring', 'sixu',
-           'integer_types', 'is_pathlib_path', 'npy_load_module', 'Path',
-           'pickle', 'contextlib_nullcontext', 'os_fspath', 'os_PathLike']
+           'integer_types', 'is_pathlib_path', 'npy_load_module',
+           'get_pickle', 'contextlib_nullcontext', 'os_fspath', 'os_PathLike']
 
 import sys
 import os
-try:
-    from pathlib import Path, PurePath
```

Comment:
Purepath is used below

Reply:
gah, there is always one! I thought I tested against the `standard` test suite. I guess there are more tests I didn't run. :(

Reply:
This file is hard to test, because you need to use every supported python version to do so.

Reply:
yes agreed. I think I addressed the concern. in the lastest force push (sorry, trying to keep each commit contained a single improvement).